### PR TITLE
reorganize test files & fix memleak of base62 encoding & small fixes 

### PR DESCRIPTION
**Pull request by Hermann0222**

### CODE REVIEW

File: nanomq/webhook_post.c [hunk 1]
```diff
@@ -62,15 +62,17 @@ set_char(char *out, unsigned int *index, char c)
...
+		// out[idx++] = 'i';
+		// out[idx++] = 'b';
+		out[idx++] = 'A';
 		break;
 	case '/':
-		out[idx++] = 'i';
-		out[idx++] = 'c';
+		// out[idx++] = 'i';
+		// out[idx++] = 'c';
+		out[idx++] = 'B';
```
Comment (wanghaEMQ) [inline nanomq/webhook_post.c:75]: I am not quite sure about it...

**Review by wanghaEMQ — APPROVED** [review]

File: nanomq/webhook_post.c (hunk 1 above)
Comment (wanghaEMQ) [reply nanomq/webhook_post.c:75]: @lee-emqx Any suggestion?

Comment (JaylinYu) [reply nanomq/webhook_post.c:75]: whats wrong here?

**Review by JaylinYu — APPROVED** [review]

File: nanomq/webhook_post.c (hunk 1 above)
Comment (Hermann0222) [reply nanomq/webhook_post.c:75]: Unpredictable usage of mem will cause over-flow

**Pull request merged**